core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -631,7 +631,8 @@ class WorkspaceServiceSpec extends FlatSpec with ScalatestRouteTest with Matcher
     }
 
     //Check method configs to be deleted exist
-    assertResult(Vector(MethodConfigurationShort("testConfig2","Sample",MethodRepoMethod("myNamespace","method-a",1),"dsde"), MethodConfigurationShort("testConfig1","Sample",MethodRepoMethod("ns-config","meth1",1),"ns"))) {
+    assertResult(Vector(MethodConfigurationShort("testConfig1","Sample",MethodRepoMethod("ns-config","meth1",1),"ns"),
```

Comment:
you can use [assertSameElements](https://github.com/broadinstitute/rawls/blob/b49619dc8400dbdf67153ea5260eea79d6d4133d/core/src/test/scala/org/broadinstitute/dsde/rawls/RawlsTestUtils.scala#L63) to sidestep ordering annoyances if they don't actually matter